test/specs/basic/ignore-joiners-synonyms.js

```diff
@@ -0,0 +1,42 @@
+/*!***************************************************
+ * mark.js
+ * https://github.com/julmot/mark.js
+ * Copyright (c) 2014–2016, Julian Motz
+ * Released under the MIT license https://git.io/vwTVl
+ *****************************************************/
+"use strict";
+describe("basic mark with ignoreJoiners synonyms", function () {
+    var $ctx1, $ctx2;
+    beforeEach(function (done) {
+        loadFixtures("basic/ignore-joiners-synonyms.html");
+
+        $ctx1 = $(".basic-ignore-joiners-synonyms > div:nth-child(1)");
+        $ctx2 = $(".basic-ignore-joiners-synonyms > div:nth-child(2)");
+        new Mark($ctx1[0]).mark("Lorem", {
+            "synonyms": {
+                "Lorem": "ipsum"
+            },
+            "separateWordSearch": false,
+            "diacritics": false,
+            "ignoreJoiners": true,
+            "done": function () {
+                new Mark($ctx2[0]).mark(["one", "dos", "lüfte"], {
+                    "separateWordSearch": false,
+                    "diacritics": false,
+                    "ignoreJoiners": true,
+                    "synonyms": {
+                        "ü": "ue",
+                        "one": "uno",
+                        "two": "dos"
+                    },
+                    "done": done
+                });
+            }
+        });
+    });
+
+    it("should wrap ignoreJoiners synonyms as well as keywords", function () {
```

Comment:
I also just realized that this file sets `diacritics` to `false` without mention... I was thinking about adding `"without diacritics"` to the end of the `describe` and `it` strings. Okay?